src/witan/cic/episodes.clj

```diff
@@ -909,13 +909,59 @@
 (defn mark-periods-that-start-before-min-report-year-xf [min-report-year]
   (map (partial mark-periods-that-start-before-min-report-year min-report-year)))
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Fix episodes that take place after the extract date
+
+(defn episode-starts-after-extract-date? [extract-date date]
```

Comment:
rename to `after-extract-date?` as it is being used more generically now. I'm happy for it to be a small function given that the nil guard is useful.

Reply:
Cool, in fact this the naming is a hangover from my first version of this fn, so a good spot 👍